spec/helpers/value_helper_spec.rb

```diff
@@ -21,9 +21,11 @@
     end
 
     it "converts time to relative time when specified" do
-      date = DateTime.new(2015,8,15,4,5,6).in_time_zone("Eastern Time (US & Canada)").strftime("%b %-d, %Y at %l:%M%P")
+      date = DateTime.new(2015,8,15,4,5,6).in_time_zone
+      date_str = date.strftime("%b %-d, %Y at %l:%M%P")
+      puts "Time.zone==#{Time.zone}"
       relative_date = time_ago_in_words(date)
-      expect(helper.date_with_tooltip(date,true)).to eq('<span title="Aug 15, 2015 at 12:05am">' + relative_date + ' ago</span>')
+      expect(helper.date_with_tooltip(date,true)).to eq('<span title="' + date_str + '">' + relative_date + ' ago</span>')
```

Comment:
I'm not sure this is testing the right thing -- as `date_with_tooltip` calls `strftime`, calling it here doesn't help us much, hey?